server/Readme.md

```diff
@@ -18,11 +16,30 @@ To test the API, call route [http://localhost:5000/values](http://localhost:5000
 
 If using Postman, deactivate SSL certificate verification.
 
+## Switching environments
+
+From a Powershell terminal, you can change environments with the following commands: 
+- `$Env:ASPNETCORE_ENVIRONMENT = "Development"`
+- `$Env:ASPNETCORE_ENVIRONMENT = "Production"`
```

Comment:
This variable appears to be required for the server to run. Is it possible to default to "Development" when the variable is not present ?

Reply:
It think it defaults to production unfortunatly

Reply:
damn...